api/src/app/modules/project/project.service.ts

```diff
@@ -238,10 +238,15 @@ export class ProjectService extends DataService<Project, Repository<Project>, Pr
     const response = new ProjectResponse();
     if (entity.commentingClosedDate) {
       response.commentingClosedDate = dayjs(entity.commentingClosedDate).format(this.DATE_FORMAT);
+      if (entity.bctsMgrName) {
```

Comment:
I'd add a comment explaining why this is necessary.

Reply:
Nice. Thanks Basil. Did it too quick and forgot it is important to add the rules for people to know.